var/spack/repos/builtin/packages/llvm-flang/package.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+import os
+
+
+class LlvmFlang(CMakePackage):
+    """LLVM-Flang is the Flang fork of LLVM needed by the Flang package."""
+
+    homepage = "https://github.com/flang-compiler"
+
+    git      = "https://github.com/flang-compiler/llvm.git"
+
+    maintainer = ['naromero77']
+
+    version('master', branch='master')
+    version('release_70', branch='release_70')
+    version('release_60', branch='release_60')
+    version('20190329', tag='flang_20190329')
+    version('20181226_70', tag='20181226_70')
+    version('20181226_60', tag='20181226_60')
+    version('20180921', tag='20180921')
+    version('20180319', tag='20180319')
+    version('20180328', tag='20180308')
+
+    # Variants
+    variant('target', default='X86',
+            description='Target archiecture',
+            values=('X86', 'PowerPC', 'AArch64'))
```

Comment:
I don't think this variant is necessary, you can pull this information from the target spec.

Reply:
Will need to dig deeper into their CMake. It only accepts three different strings for `target` -- it may be as simple as each `target` CMake option injects different compiler flags -- in which case, I could just delete this variant and use the builtin. If their CMake does something more than just inject compiler flags, then maybe I will need to keep this variant.

Reply:
I meant you can pull this information from the target spec _within the `cmake_args` function_. @alalazo or @becker33 can point you to the right syntax, but basically you'll just be checking if the requested target is x86, PowerPC, or ARM.